search-app/docs/api.md

```diff
@@ -1435,6 +1435,18 @@ Find collections matching measurement.
 
     curl "%CMR-ENDPOINT%/collections?measurement\[\]=Ozone&measurement\[\]=radiance"
 
+#### <a name="c-variables"></a> Find collections by variables
+
+This supports option _or_.
+
+Find collections matching 'variables' param value
+
+     curl "%CMR-ENDPOINT%/collections?variables\[0\]\[variable\]=Var1"
+
+Find collections matching multiple 'variables' param values, default is :and
+
+     curl "%CMR-ENDPOINT%/collections?variables\[0\]\[variable\]=Var1&variables\[0\]\[measurement\]=M1&variables\[1\]\[variable\]=Var2"
```

Comment:
Do we support searches for variables[0][variable]? I see the facets are using variables_h[0][variable].